queue_job/readme/USAGE.rst

```diff
@@ -274,6 +274,23 @@ Based on this configuration, we can tell that:
 * retries 10 to 15 postponed 30 seconds later
 * all subsequent retries postponed 5 minutes later
 
+**Job function: Error Handler**
+
+The *Error Handler* is actions executed when job fails
+
+It's configured similarly to Related Action
+
+There is an OOTB handler: _call_api, which send a webhook with configurable information.
+
+The webhook URL is configured in System parameters via key **queue_job.webhook_url**
+
+Example of using _call_api:
```

Comment:
I think you can give the slack example to make it even more appealing/useful out of the box (with a webhook url that looks real but with tokens that you manually changed to XXX...)